packages/data/walkthroughs/9.9.9.test.ts

```diff
@@ -0,0 +1,46 @@
+// 3rd party
+import { describe, it, expect } from "vitest";
+// local
+import { isWalkthroughItemTypeVariable } from "../src/useWalkthroughData";
+import data from "../walkthroughs/9.9.9.json";
+
+describe("Data - 9.9.9", () => {
+  it("verify all questions either appear in a section or are of variable type", () => {
+    // iterate through each question and check if it appears in a section or is of variable type
+    const possibleUnusedQuestion = Object.entries(data.questions).find(
+      ([questionId, question]) => {
+        if (isWalkthroughItemTypeVariable(question.walkthroughItemType)) {
+          return false;
+        }
+
+        return !Object.values(data.sections).some((section) =>
+          section.sectionQuestions.includes(questionId),
+        );
+      },
+    );
+    if (possibleUnusedQuestion)
+      console.log("Possible unused question ID:", possibleUnusedQuestion[0]);
```

Comment:
Do we need this console log in the test? Won't the expect tell us the value of `possibleUnusedQuestion` if the test fails?

Reply:
It does, but it's not as easy to read/readily apparent IMO. This was just for readability.